tests/test_derivative_symbols.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
Why is this a separate file? Would expect this in test_model.py or something that relates to the class name

Reply:
because yo complained when i put the test for state_symbols in the model and told me to make a seperate est. this is the same thing so also a separete test for consistancy!

Reply:
Can we at least call it something with "model" in the filename? Need to be able to find these tests later

Reply:
Hhm then it would be better to move it into the model. I don't mind that if others don't mind.

Reply:
I wouldn't mind that. But if you say we've discussed it before.. I can't find that discussion though!

Reply:
Probably good to see what reasons / arguments / conclusions we had :D

Reply:
I can't find it sorry

Reply:
I can't find it sorry